ote/src/cljs/ote/views/own_services.cljs

```diff
@@ -71,36 +71,38 @@
-       (fn [i {::t-service/keys [id type published? name]
+       (fn [i {::t-service/keys [id type published name]
                ::modification/keys [created modified] :as row}]
          ^{:key i}
-         [ui/table-row {:selectable false :display-border false}
-          [ui/table-row-column
-           [:a (merge {:href (str "/#/edit-service/" id)
-                       :on-click #(do
-                                    (.preventDefault %)
-                                    (e! (fp/->ChangePage :edit-service {:id id})))}
-                      (stylefy/use-sub-style style-base/front-page-service-table :link)) name]]
-          [ui/table-row-column {:class "hidden-xs "} (tr [:field-labels :transport-service ::t-service/published?-values published?])]
-          [ui/table-row-column {:class "hidden-xs hidden-sm "} (time/format-timestamp-for-ui modified)]
-          [ui/table-row-column {:class "hidden-xs hidden-sm "} (time/format-timestamp-for-ui created)]
-          [ui/table-row-column {:class "hidden-xs hidden-sm "}
-           (if published?
-             (let [url (str "/export/geojson/" transport-operator-id "/" id)]
-               [linkify url
-                (tr [:own-services-page :open-geojson])
-                {:target "_blank"
-                 :style {:text-decoration "none"
-                         ::stylefy/mode {:hover {:text-decoration "underline"}}}}])
-             [:span.draft
-              (tr [:field-labels :transport-service ::t-service/published?-values false])])]
-          [ui/table-row-column
-           [ui/icon-button (merge {:href "#" :on-click #(do
-                                                          (.preventDefault %)
-                                                          (e! (fp/->ChangePage :edit-service {:id id})))}
-                                  (stylefy/use-style {::stylefy/manual [[:&:hover [:svg {:color (str colors/primary " !important")}]]]}))
-            [ic/content-create]]
-           [delete-service-action e! row]]])
+         (do
+           (.log js/console published)
+           [ui/table-row {:selectable false :display-border false}
+            [ui/table-row-column
+             [:a (merge {:href (str "/#/edit-service/" id)
+                         :on-click #(do
+                                      (.preventDefault %)
+                                      (e! (fp/->ChangePage :edit-service {:id id})))}
+                        (stylefy/use-sub-style style-base/front-page-service-table :link)) name]]
+            [ui/table-row-column {:class "hidden-xs "} (tr [:field-labels :transport-service ::t-service/published?-values (not (nil? published))])]
+            [ui/table-row-column {:class "hidden-xs hidden-sm "} (time/format-timestamp-for-ui modified)]
+            [ui/table-row-column {:class "hidden-xs hidden-sm "} (time/format-timestamp-for-ui created)]
+            [ui/table-row-column {:class "hidden-xs hidden-sm "}
+             (if published
```

Comment:
Stupid question, but just double checking was this tested to ensure db `NULL` converts properly to clj `nil` so that `if` is evaluated correctly?